### PR TITLE
Button styling

### PR DESCRIPTION
**Pull request by epachols**

### CODE REVIEW

File: src/webapp/features/rabble/GameScreen.tsx [hunk 1]
```diff
@@ -34,7 +34,9 @@ const GameScreen = (props: GameBoardProps) => {
   return (
     <div className={styles.board}>
       <div className={styles.topButtonContainer}>
-        <InviteLink gameID={gameID} />
+        {scoreList[1] && !scoreList[1].nickname.length ? (
```
Comment (apachols) [inline src/webapp/features/rabble/GameScreen.tsx]: Can we also add an `Array.isArray(scoreList)` check here?  it'll prevent a crash if scoreList is for some reason undefined.

I bet we could use optional chaining too:
```
Array.isArray(scoreList) && scoreList[1]?.nickname?.length ? null : <InviteLink>
```

Or something similar

Comment (epachols) [reply src/webapp/features/rabble/GameScreen.tsx]: got it !

File: src/webapp/features/rabble/GameScreen.tsx [hunk 2]
```diff
@@ -34,7 +34,9 @@ const GameScreen = (props: GameBoardProps) => {
   return (
     <div className={styles.board}>
       <div className={styles.topButtonContainer}>
-        <InviteLink gameID={gameID} />
+        {typeof scoreList === "object" && !scoreList[1]?.nickname?.length ? (
```
Comment (apachols) [inline src/webapp/features/rabble/GameScreen.tsx:37]: Ah cool, i like the brevity in the second term in this conditional 👍 

interesting to check `object` here, i guess any object would be able to be derefrenced with square brackets!

Comment (apachols) [reply src/webapp/features/rabble/GameScreen.tsx:37]: This makes me wonder whether it's best practice to put these old javascripty null checks in typescript code... Let's keep it like it is for now.  Good job cleaning it up 💯

**Review by apachols — APPROVED** [review]
LGTM!  Dark mode looks great = )

**Pull request merged**